docs/devguide/contributing.rst

```diff
@@ -35,3 +35,86 @@ negative votes, the team will grant commit privileges.
 The core team will also be responsible for removing commit privileges when
 appropriate - for example for malicious merge behavior or just inactivity over
 an extended period of time.
+
+
+Quality gates for the pull requests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are three quality gates to ensure the pull requests quality, `Hound`_ for
+code style check, `Travis CI`_ for unit-test and coveralls, `Jenkins`_ for the combination 
+test including unit-test and smoke test.. When a pull request created, all tests 
```

Comment:
An extra period after "smoke test" shouldn't be needed and also suggest reword to "When a pull request is created, all tests will run automatically and the test results can be found .....").